.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Alternatively, if you let me know what the credentials look like, I can whip up a quick pre-commit hook to prevent checkins of credentials.